lib/fireworq.go

```diff
@@ -253,6 +279,13 @@ func Do() {
 	}
 	fireworq.LabelPrefix = *optLabelPrefix
 
+	rawQueueStats := strings.Split(strings.Join(strings.Fields(*optQueueStats), ""), ",")
+	stats := make([]QueueStat, 0)
```

Comment:
We know the size of `stats`.

```suggestion
	stats := make([]QueueStat, 0, len(rawQueueStats))
```